includes/Query.php

```diff
@@ -62,7 +62,7 @@ public function fetch() {
 		case static::FILTER_REPORTED:
 			$cond['flowthread_status'] = Post::STATUS_NORMAL;
 			$cond[] = 'flowthread_report > 0';
-			$options['ORDER BY'] = 'flowthread_report DESC, ' . $options['ORDER BY'];
+			$options['ORDER BY'] .= ', flowthread_report DESC';
```

Comment:
`flowthread_id` will never duplicate, so additional order condition is meaningless.

Reply:
OK, delete it